packages/cli/test/integration/workflows.controller.ee.test.ts

```diff
@@ -0,0 +1,117 @@
+import express from 'express';
+
+import * as utils from './shared/utils';
+import * as testDb from './shared/testDb';
+import { createWorkflow } from './shared/testDb';
+import * as WorkflowHelpers from '../../src/workflows/helpers';
+
+import type { Role } from '../../src/databases/entities/Role';
+
+jest.mock('../../src/telemetry');
+
+// mock whether workflow sharing is enabled or not
+const mockIsWorkflowSharingEnabled = jest.spyOn(
+	WorkflowHelpers,
+	'isWorkflowSharingEnabled',
+);
+mockIsWorkflowSharingEnabled.mockReturnValue(true);
```

Comment:
Can we do this inside the tests to avoid shared state?

Reply:
But this is necessary for all tests so that the controller registers correctly - the code is identical to the credential tests.